ci/jenkins/test.sh

```diff
@@ -620,6 +619,9 @@ function deliver_antrea {
         scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" build/yamls/*.yml jenkins@[${control_plane_ip}]:~
```

Comment:
I saw this line use `[${control_plane_ip}]` directly, can this work for both ipv4 and ipv6? if yes, I feel there is unnecessary to check the control_plane_ip type in following code. Could you check?

Reply:
I have tried to run `ssh jenkins$[$IP]` manually, but got an error shows `Could not resolve hostname`. I'll double check this part, thanks .

Reply:
After testing in `bash`, the commad `scp -i key USER@[IP]` works well, so the if condition could be removed.